### DISCUSSION TITLE
New Styles and Thumbnails!

### DISCUSSION THREAD

**Discussion by 3Diva** [lllyasviel/Fooocus#1860] [General]
I don't know if this is posted somewhere (if so please share the link) but it appears that Fooocus has had a LOT of new styles added to it (62 new styles I believe), and when you hover over the styles it also shows a preview thumbnail of what the style looks like. This is an INCREDIBLE update and SO HELPFUL! I love the new styles and the great preview images when you hover over the styles. Thank you so much, @lllyasviel - This is AWESOME!

**Comment by eddyizm** [comment]
a very neat feature indeed!

**Comment by poor7** [comment]
v2.1.853 Add Marc K3nt3L's styles. Thanks [Marc K3nt3L](https://github.com/K3nt3L)!
https://github.com/lllyasviel/Fooocus/discussions/1378

**Reply by 3Diva** [reply]
@poor7 Oh thank you for the link - I missed that discussion! :) The thumbnails are a great touch too. So helpful!

**Comment by mashb1t** [comment]
We only have the feature due to @crohrer implementing it in the first place 🎉
